CS446Project/app/src/main/java/com/example/qian/cs446project/PlaylistParcelable.java

```diff
@@ -0,0 +1,50 @@
+package com.example.qian.cs446project;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * Created by Qian on 2018-02-24.
+ */
+
+// Ke Qiao Chen: I based this class on
+// http://www.vogella.com/tutorials/AndroidParcelable/article.html
+public class PlaylistParcelable implements Parcelable {
```

Comment:
Can we just apply this to the existing Playlist class instead of creating a new one?